### PR TITLE
Symfony 7 support

### LINKED ISSUE #128: Symfony 7 support

### PR DESCRIPTION
**Pull request by radimvaculik**
Closes  #128

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	"type": "library",
 	"license": "BSD-3-Clause",
 	"require": {
-		"php": ">=5.4"
+		"php": ">=7.0"
```
Comment (hrach) [inline composer.json:7]: I'm all in but not sure if @JanTvrdik - he was specifically pushing this.

Comment (radimvaculik) [reply composer.json:7]: Fully understand.

Comment (JanTvrdik) [reply composer.json:7]: I have mostly ready [upgrade to PHP 7.1](https://github.com/nextras/migrations/pull/130) baseline. I hope to finish that, merge this and release on weekend.

Comment (radimvaculik) [reply composer.json:7]: OK, closing this issue.

Comment (hrach) [reply composer.json:7]: It seems that it is about PHP 7.1, not Symfony 7 support.

**Pull request closed without merging**